taxi_service/settings.py

```diff
@@ -12,6 +12,8 @@
 
 from pathlib import Path
 
+from django.template.context_processors import static
```

Comment:
Please review how to handle static files [here](https://docs.djangoproject.com/en/5.0/howto/static-files/)